### DISCUSSION TITLE
✅ ssh-copy-id

### DISCUSSION THREAD

**Discussion by haciendasanpedro** [isontheline/pro.webssh.net#616] [General]
Greetings,

Is the ssh-copy-id command available?  I need to send my key to a new machine but don't seem to be able to reach a blank terminal window to run the command and send the key to the new machine.

Great app!  Thanks for all your hard work!

**Comment by isontheline** [comment]
Hello @haciendasanpedro 👋

Thank you a lot for your support about WebSSH 🙏

Right now ssh-copy-id isn't implemented.
I have just added your request to this issue in order to track progress : https://github.com/isontheline/pro.webssh.net/issues/46

**Reply by haciendasanpedro** [reply]
perfect, thank you!

**Reply by isontheline** [reply]
Hello @haciendasanpedro 👋

I don't know if you still use WebSSH but I'm working on it right now.
Should be available soon.
